source/WorkingPractices/TestSuites/lfric_apps.rst

```diff
@@ -3,14 +3,22 @@
 Testing LFRic Apps
 ==================
 
-LFRic Apps testing is run with the following commands from a working copy:
+Rose stem:
+    LFRic Apps testing uses rose-stem and is run with the following commands
+    from a working copy:
 
-.. code-block::
+    .. code-block::
+
+        export CYLC_VERSION=8
+        rose stem --group=developer
+        cylc play <working copy name>
+        cylc gui
+
+Local testing:
```

Comment:
Worth emphasising this is a basic command line-based test that doesn't use rose & cylc?

Reply:
done